experimental_docker_compose_files/00_old_one_from_proposal/README.md

```diff
@@ -0,0 +1,8 @@
+## Example 0
+### Old docker-compose file 
+* This is the docker-compose file I created for the proposal, which mimics the steps from the docker section of the Jenkins docker page.
+* This file does not configure any agent.
+* More info about the file can be found in the proposal [here](https://docs.google.com/document/d/1ZpPihadYqpAvR20rxZkTD2SVpf34E6YMzg6opU6yHAg/edit#heading=h.lntg56ljm653).
+### How to run 
+* Make sure you are in the `00_old_one_from_proposal` directory by using the command `pwd`. 
+* Then use the command `docker compose up -d` to run the example.
```

Comment:
This doesn't work on my machine (at least as long as the compose file is prefixed with 00). Machine is Mac OS with  Docker 20.10.10.
```
 01_simple_controller+agent> docker compose up
can't find a suitable configuration file in this directory or any parent: not found
```

To make it work, I need to specify the `-f` with the exact name of the compose file.. We could either rename the compose file or update the documentation.

I vote for renaming the compose file.

Reply:
I've renamed the files

Reply:
I think we can mention this issue: https://github.com/ash-sxn/GSoC-2023-docker-based-quickstart/issues/21